app/editor/components/CommandMenu.tsx

```diff
@@ -80,14 +83,6 @@ class CommandMenu<T extends MenuItem> extends React.Component<Props<T>, State> {
     window.addEventListener("keydown", this.handleKeyDown);
   }
 
-  shouldComponentUpdate(nextProps: Props<T>, nextState: State) {
-    return (
-      nextProps.search !== this.props.search ||
-      nextProps.isActive !== this.props.isActive ||
-      nextState !== this.state
-    );
-  }
-
```

Comment:
Removed `shouldComponentUpdate` to fix :point_down:
> Menu items are not updated immediately but updated after next keystroke - off by one issue.

`shouldComponentUpdate` seemed to [prevent re-rendering when necessary](https://reactjs.org/docs/react-component.html#shouldcomponentupdate).

Hoping that the removal isn't a problem :crossed_fingers:

Reply:
This is pretty dated stuff, I'm not too worried about it